mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -98,6 +98,15 @@ class LogisticRegression(override val uid: String)
   def setFitIntercept(value: Boolean): this.type = set(fitIntercept, value)
   setDefault(fitIntercept -> true)
 
+  /**
+   * Whether to standardize the training features prior to fitting the model sequence.
```

Comment:
"model sequence" may not be understood since we don't provide a sequence currently; how about just saying "model?"

Also, to make it clear what is meant by standardizing, how about having a link to StandardScaler.withStd?